core/iwasm/aot/aot_runtime.h

```diff
@@ -109,6 +109,7 @@ typedef struct AOTModuleInstanceExtra {
 #if WASM_ENABLE_MULTI_MODULE != 0
     bh_list sub_module_inst_list_head;
     bh_list *sub_module_inst_list;
+    WASMModuleInstanceCommon **func_module_insts;
```

Comment:
How about renaming to `import_func_module_insts` since we should only need to create module instance for import functions?